### PR TITLE
Default to https if no scheme specified when using api_host

### LINKED ISSUE #6: api host should assume https if the protocol is absent
Invoking honeyvent with `--api_host example.com` yields an error `unsupported protocol scheme ""]`. Instead if the address passed to `api_host` is missing a protocol, HTTPS should be assumed.

### PR DESCRIPTION
**Pull request by paulosman**
Fixes #6.

### CODE REVIEW

**Review by vreynolds — APPROVED** [review]
🔒

File: main_test.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+)
+
+func TestAPIHostParsing(t *testing.T) {
+	u, err := parseAPIHost("api.honeycomb.io")
+	if err != nil {
+		t.Error(err)
+	}
+	if u.Scheme != "https" {
+		t.Errorf("Expected https, got: %s", u.Scheme)
+	}
```
Comment (robbkidd) [inline main_test.go:14]: Would it be appropriate to add another test case for when a scheme is provided in the URL to parse?

```go
u, err := parseAPIHost("http://some.onprem.test.host.without.a.cert.example.com"")
if err != nil {
	t.Error(err)
}
if u.Scheme != "http" {
	t.Errorf("Expected http, got: %s", u.Scheme)
}
```

Comment (paulosman) [reply main_test.go:14]: I dig it. Added!

**Review by robbkidd — COMMENTED** [review]
a post-hoc 👍

**Pull request merged**